.claude/skills/beopsuny/scripts/generate_ical.py

```
@@ -0,0 +1,374 @@
+def generate_uid(item_id: str, year: int) -> str:
+    """고유 UID 생성"""
+    return f"{item_id}-{year}@beopsuny.legal-stack"
+
+
+def create_vevent(
+    uid: str,
+    summary: str,
+    dtstart: datetime,
+    description: str = "",
+    location: str = "",
+    categories: List[str] = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)